### PR TITLE
Upgraded gems, added more versions to .travis.yml file, fixed typo

### PR DESCRIPTION
**Pull request by jasnow**
* Upgraded gems.
 * Added more versions to .travis.yml file.
 * Fixed typo.

### CODE REVIEW

**Comment by djberg96** [comment]
@jasnow I don't suppose you could squash all those commits, could you?

**Comment by jasnow** [comment]
Never used it before - will need some time to learn.

**Comment by jasnow** [comment]
@djberg96 - Done

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,19 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    power_assert (1.1.0)
```
Comment (djberg96) [inline Gemfile.lock]: What's power_assert? I don't require that.

Comment (jasnow) [reply Gemfile.lock]: It appears that "test-unit >= 3.0" needs it: https://github.com/k-tsj/power_assert and https://rubygems.org/gems/test-unit

**Comment by jasnow** [comment]
Thanks

**Pull request merged**